it/connectors/CreateOrAmendSavingsTysConnectorISpec.scala

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import com.github.tomakehurst.wiremock.http.HttpHeader
+import config.{AppConfig, BackendAppConfig}
+import helpers.WiremockSpec
+import models._
+import org.scalatestplus.play.PlaySpec
+import play.api.Configuration
+import play.api.http.Status._
+import play.api.libs.json.{JsObject, Json}
+import uk.gov.hmrc.http.{HeaderCarrier, HeaderNames, HttpClient, SessionId}
+import uk.gov.hmrc.play.bootstrap.config.ServicesConfig
+import utils.TaxYearUtils.convertSpecificTaxYear
+
+class CreateOrAmendSavingsTysConnectorISpec extends PlaySpec with WiremockSpec {
+
+  val model: CreateOrAmendSavingsModel = CreateOrAmendSavingsModel(
+    securities = SecuritiesModel(Some(800.67), 7455.99, Some(6123.2)),
+    foreignInterest = Seq(ForeignInterestModel("BES", Some(1232.56), Some(3422.22), Some(5622.67), Some(true), 2821.92))
+  )
+
+  implicit val hc: HeaderCarrier = HeaderCarrier()
+  lazy val connector: CreateOrAmendSavingsTysConnector = app.injector.instanceOf[CreateOrAmendSavingsTysConnector]
+  lazy val httpClient: HttpClient = app.injector.instanceOf[HttpClient]
+
+  def mkTaxYear(taxYear: Int): String = s"${taxYear - 1}-${taxYear.toString takeRight 2}"
+
+  val nino = "nino"
+  val taxYear = 2024
```

Comment:
read from util

Reply:
Yep, made this change in the next commit